src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const ourAcrion of actions) {
```

Comment:
our? are you from the soviet union?
```suggestion
  for (const action of actions) {
```